CONTRIBUTING.md

```diff
@@ -4,6 +4,24 @@ This document should concisely express the project development status,
 methodology, and contribution process.  As the community makes progress, we
 should keep this document in sync with reality.
 
+## Submitting a Pull Request (PR)
+
+The following outlines the general rules we follow:
+
+- All PRs must have the appropriate documentation changes made within the
```

Comment:
@pmorie if our CI really does take 2hrs then I would suggest people open 2 PRs at the same time - one for code and one for docs - but I wouldn't want to encourage that until our CI really is that slow.